crates/workspace-hack/Cargo.toml

```diff
@@ -14,7 +14,7 @@ publish = false
 # are managed by hakari.
 
 ### BEGIN HAKARI SECTION
-[dependencies]
+[target.'cfg(not(windows))'.dependencies]
```

Comment:
I'll look into adding a feature to hakari on Monday to do this via config so we don't have to worry about doing it by hand.